groot/routes.py

```diff
@@ -1,17 +1,52 @@
 from flask import Flask, Response, render_template, request, redirect, url_for, flash
 from run import app, db
-
+from groot.models import *
+import hashlib
 
 
 @app.route("/")
 @app.route('/login', methods=['POST', 'GET'])
 def login():
-    return render_template('login.html')
-        
+    if request.method == 'POST':
```

Comment:
Before this, make sure the user is not already signed in.

Reply:
NVM since you didn't implement actual login yet, but remember to do it later

Reply:
we will use later on the login_required decorator in flask login.